src/tests/suite/dfs_test.c

```diff
@@ -179,13 +178,10 @@ dfs_test_read_thread(void *arg)
 		read_size = min(targ->total_size - off, targ->stride);
 		sgl.sg_iovs[0].iov_len = read_size;
 
-		if (count % 10 == 0)
-		print_message("thread %d try to read off %d, size %d......\n",
-			      targ->thread_idx, (int)off, (int)read_size);
```

Comment:
minor, sometimes I enlarge the file_size and threads_num for manual testing, without the msg above the test just looks like hang.